static/js/containers/pages/CheckoutPage.js

```diff
@@ -170,6 +170,7 @@ export class CheckoutPage extends React.Component<Props, State> {
           dataLayer.push({
             event:            "purchase",
             transactionId:    payload.transaction_id,
+            coursePrice:      payload.item_0_unit_price || "0",
```

Comment:
This looks different than `course-price` as requested on the ticket. Also, please note that `coursewareId` is being pushed, not `course-id` (which you might want to add alongside).

Reply:
what do you think, `course-price` and `course-id` would be good key literals in Javascript ?

Reply:
You can put the key in quotes and it will work. JS does not have a limitation on keys with dashes.